Completely ignore m2r (which is now last-rited) so we can drop the hard-blocker
which makes world upgrades a pain (installed m2r would break b/c of newer
mistune, and would break automat build).

https://github.com/miyakogi/m2r/issues/66
https://bugs.gentoo.org/848900
https://bugs.gentoo.org/847532
--- a/setup.py
+++ b/setup.py
@@ -4,13 +4,8 @@ Setup file for automat
 
 from setuptools import setup, find_packages
 
-try:
-    from m2r import parse_from_file
-    long_description = parse_from_file('README.md')
-except(IOError, ImportError):
-    print("\n\n!!! m2r not found, long_description is bad, don't upload this to PyPI !!!\n\n")
-    import io
-    long_description = io.open('README.md', encoding="utf-8").read()
+import io
+long_description = io.open('README.md', encoding="utf-8").read()
 
 setup(
     name='Automat',
